pennylane_lightning/core/src/utils/cuda_utils/cuGates_host.hpp

```diff
@@ -653,17 +654,25 @@ template <class CFP_t, class U = double>
 static auto getSingleExcitation(U angle) -> std::vector<CFP_t> {
     const U p2 = angle / 2;
     const CFP_t c{std::cos(p2), 0};
-    const CFP_t s{std::sin(p2), 0};
+    // TODO: To remove conditional compilation here in the future, current
+    // implementation will block the simultaneous installation of LGPU and
+    // cutensornet backends
+
+#ifdef _ENABLE_PLGPU
+    const CFP_t s{-std::sin(p2), 0}; // column-major
+#else
+    const CFP_t s{std::sin(p2), 0}; // row-major
+#endif
     return {cuUtil::ONE<CFP_t>(),
             cuUtil::ZERO<CFP_t>(),
             cuUtil::ZERO<CFP_t>(),
             cuUtil::ZERO<CFP_t>(),
             cuUtil::ZERO<CFP_t>(),
             c,
-            s,
+            -s,
             cuUtil::ZERO<CFP_t>(),
             cuUtil::ZERO<CFP_t>(),
-            -s,
+            s,
```

Comment:
Is this change correct? Why so?

Reply:
it is correct and all unit tests passed. I just revert it to make it aligned with `Gates.hpp`. col-major layout is done with the conditional compilation.

Reply:
So this data is now fully in line with LightningQubit, as well as default.qubit?
And only uses col major when requested ?

Reply:
yes, for the moment, only lightning.gpu requires col-major memory layout.